R/mutate.R

```diff
@@ -0,0 +1,162 @@
+#' Add new variables using `mutate`
+#'
+#' `step_mutate` creates a *specification* of a recipe step
+#'  that will add variables using [dplyr::mutate()].
+#'
+#' @inheritParams step_center
+#' @param ... Name-value pairs of expressions. See [dplyr::mutate()].
+#' @param role For model terms created by this step, what analysis
+#'  role should they be assigned? By default, the function assumes
+#'  that the new dimension columns created by the original variables
+#'  will be used as predictors in a model.
+#' @param input Quosure(s) of `...`.
+#' @return An updated version of `recipe` with the new step
+#'  added to the sequence of existing steps (if any). For the
+#'  `tidy` method, a tibble with columns `values` which
```

Comment:
Maybe mention that the expressions in `values` (strange name for that column, but I guess that's consistent with other tidy methods?) are not reparsable because the quosures have been squashed, and you couldn't represent the environment as string anyway. These deparsed expressions are purely indicative.